src/pages/send/choose-fee-level/choose-fee-level.html

```diff
@@ -41,15 +41,15 @@
         <ion-note item-end>
           <span *ngIf="loadingFee">...</span>
           <span *ngIf="feePerSatByte && !loadingFee">
-            {{feePerSatByte}} sat/byte
+            {{feePerSatByte}} {{ feeUnit }}
           </span>
         </ion-note>
       </ion-item>
     </ion-list>
 
     <ion-list class="fee-custom" *ngIf="customFeePerKB">
       <ion-item>
-        <ion-label color="primary" floating>{{'Enter custom fee in sat/byte' | translate}}</ion-label>
+        <ion-label color="primary" floating>{{'Enter custom fee in ' + feeUnit | translate}}</ion-label>
```

Comment:
Check this translate, I don't think it works as expected

Reply:
fixed

@Gamboster What do you think about doing it this way?:
```
{{'Enter custom fee in {feeUnit}' | translate: { feeUnit: feeUnit } }}
```